Frameworks/CoreText/CTFrame.mm

```diff
@@ -55,7 +55,7 @@ CFRange CTFrameGetVisibleStringRange(CTFrameRef frame) {
     _CTFrame* framePtr = static_cast<_CTFrame*>(frame);
     CFIndex count = 0;
     if (framePtr) {
-        for (CFIndex i = 0; i < framePtr->_lineOrigins.size() && framePtr->_lineOrigins[i].y > 0; ++i) {
+        for (CFIndex i = 0; i < [framePtr->_lines count] && static_cast<_CTLine*>(framePtr->_lines[i])->_lineOrigin.y >= 0; ++i) {
```

Comment:
0 would be the bottom of the frame, so any line with an origin >= 0 would be completely within the frame

---
In reply to: [78260477](https://github.com/Microsoft/WinObjC/pull/989#discussion_r78260477) [](ancestors = 78260477)